clicommand/agent_start.go

```diff
@@ -769,6 +776,7 @@ var AgentStartCommand = cli.Command{
 			DisconnectAfterIdleTimeout: cfg.DisconnectAfterIdleTimeout,
 			CancelGracePeriod:          cfg.CancelGracePeriod,
 			EnableJobLogTmpfile:        cfg.EnableJobLogTmpfile,
+			JobLogTmpDirPath:           cfg.JobLogTmpDirPath,
 			Shell:                      cfg.Shell,
 			RedactedVars:               cfg.RedactedVars,
 			AcquireJob:                 cfg.AcquireJob,
```

Comment:
After testing this a bit more, it might only work for long running jobs with our log collector. I wonder how yall would feel if we just exposed a different flag that had the agents write all job logs to a persistent file (which could be rotated by the system)?